Assets/Scripts/Model/Director.cs

```diff
@@ -103,6 +169,8 @@ public bool AddMaterial(int index)
                     Fabrics[index].RemoveMat();
             }
             Money -= Fabrics[index].ProcPrice;
+            if (ret)
```

Comment:
`ret` is a confusing name for me. Does it come from `return`? A better option could be `isMaterialAdded`, for example.